e2e/baseFixtures.js

```diff
@@ -103,25 +103,40 @@ const extendedTest = test.extend({
    * Default: `true`
    */
   failOnConsoleError: [true, { option: true }],
+  ignore404s: [[], { option: true }],
   /**
    * Extends the base page class to enable console log error detection.
    * @see {@link https://github.com/microsoft/playwright/discussions/11690 Github Discussion}
    */
-  page: async ({ page, failOnConsoleError }, use) => {
+  page: async ({ page, failOnConsoleError, ignore404s }, use) => {
     // Capture any console errors during test execution
-    const messages = [];
+    let messages = [];
     page.on('console', (msg) => messages.push(msg));
-
+    
     await use(page);
 
+    if (ignore404s.length > 0){
```

Comment:
you can simplify this to `if(ignore404s.length) ... cont`

Reply:
Thanks for the feedback! You're absolutely right, I could use truthiness here and make the code more succinct. As a general rule I don't like to rely on truthiness though, I try to be explicit about intent. When truthiness fails it can cause bugs that are difficult to debug. I try to write codes for idiots, because we're all idiots sometimes :)